app/views/categories/_form.html.erb

```diff
@@ -55,5 +55,12 @@
     <span class="form-caption">Tick this box to set this category as the site homepage.</span>
   </div>
 
+  <div class="form-group">
+    <%= f.label :color_code, 'Color (in the header)', class: 'form-element' %>
+    <span class="form-caption">Select a color, which shall be used for the header of the views within this category. Leave empty for site-wide default.</span>
+    <%= f.select :color_code, options_for_select(['turqoise', 'green', 'blue', 'purple', 'gray', 'yellow', 'orange', 'pink', 'red'], selected: @category.color_code),
```

Comment:
Typo in `turquoise`

Reply:
Oh. haha. Problem: It's wrong in Co-Design. I'll fix it later there and then here.

Reply:
Uhm. I seem to have forgotten to build that change. This will be fixed in the next version of Co-Design, but I'll leave this for now (in this PR, to be fixed later)